.github/styles/config/vocabularies/Base/accept.txt

```diff
@@ -173,8 +173,14 @@ Vuejs
 websockets?
 wireframes?
 workspace
-yaml
+yaml|YAML
```

Comment:
I'm off the opinion of being opiniated and when both are accepted only tolerate one.